lib/plug/debugger.ex

```diff
@@ -122,7 +122,23 @@ defmodule Plug.Debugger do
 
       def call(conn, opts) do
         try do
-          super(conn, opts)
+          case conn do
+            %Plug.Conn{path_info: ["__debug__", "action"], params: params} ->
+              %Plug.Conn{params: params} = Plug.Parsers.call(conn, Plug.Parsers.init(parsers: [:urlencoded]))
+
+              {module, function, args} = 
+                params["encoded_action"] 
+                |> Base.decode64!() 
+                |> :erlang.binary_to_term()
+                
+              apply(module, function, args)
+
+              conn
+              |> Plug.Conn.put_resp_header("location", params["last_path"])
+              |> send_resp(302, ~s|<html><body>You are being <a href="#{params["last_path"]}">redirected</a>.</body></html>|)
```

Comment:
it is not good practice to have a lot of code inside quoted code. So you can put the code of this branch on a function and call it "Plug.Debugger.handle_action(conn)".